### PR TITLE
Update setup.sql

### PR DESCRIPTION
**Pull request by nhokzminh698**
- Added business information table

### CODE REVIEW

File: setup.sql [hunk 1]
```diff
@@ -44,4 +44,12 @@ CREATE TABLE account (
 	first_name TEXT NOT NULL,
 	last_name TEXT NOT NULL,
 	account_type TEXT NOT NULL
-)
+);
+
+CREATE TABLE business_information (
+	id SERIAL PRIMARY KEY,
+	business_name VARCHAR(50) NOT NULL,
+	owner_id INT NOT NULL,
```
Comment (gitnamehere) [inline setup.sql:52]: i don't think we need an owner_id, but we could add in a phone number and address column here

**Review by mhxynh — APPROVED** [review]

File: setup.sql (hunk 1 above)
Comment (nhokzminh698) [reply setup.sql]: Okay added!

File: setup.sql [hunk 2]
```diff
@@ -44,4 +44,12 @@ CREATE TABLE account (
...
 	account_type TEXT NOT NULL
-)
+);
+
+CREATE TABLE business_information (
+	id SERIAL PRIMARY KEY,
+	business_name VARCHAR(50) NOT NULL,
+	phone_number VARCHAR(15) NOT NULL, --Phone example: 1-800-123-456
+	address TEXT NOT NULL,
+	description TEXT NOT NULL,
```
Comment (gitnamehere) [inline setup.sql:54]: dont forget to remove the last comma
```suggestion
	description TEXT NOT NULL
```

**Comment by nhokzminh698** [comment]
I committed your suggestion!

**Review by gitnamehere — APPROVED** [review]

**Pull request merged**